docs/02-nbuild.targets.md

```diff
@@ -0,0 +1,12 @@
+# Add nbuild.targets
+
```

Comment:
@naz-hage is this enough for the instructions since we will be improving `ntools` docs?

Reply:
It's enough for now. Once the project is complete, we review and make improvements